scb-engine/src/main/java/io/securecodebox/engine/service/AuthService.java

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ *  SecureCodeBox (SCB)
+ *  Copyright 2015-2018 iteratec GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ * /
+ */
+package io.securecodebox.engine.service;
+
+import io.securecodebox.engine.model.PermissionType;
+import io.securecodebox.engine.model.ResourceType;
+import org.camunda.bpm.engine.identity.Group;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.authentication.InsufficientAuthenticationException;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AuthService {
+    private static String AUTH_DISABLED_TYPE = "none";
+    private static final Logger LOG = LoggerFactory.getLogger(AuthService.class);
+
+    @Autowired
+    ProcessEngine engine;
+
+    @Value("${securecodebox.rest.auth}")
+    private String authType;
+
+    public void isAuthorizedFor(String resourceId, ResourceType resource, PermissionType permission) throws InsufficientAuthenticationException{
```

Comment:
I would name this "checkAuthorization". From a mehtod "isAuthorized" I would expect that it returns a boolean